Solutions/DNS Essentials/Analytic Rules/DNSRequestToKnownDomainIOCs.yaml

```diff
@@ -1,7 +1,7 @@
 id: 8e4a010a-972c-4124-8c27-1efcd7e3125a
 name: Detect DNS request to known Domain IOCs (ASIM DNS Solution)
 description: |
-  "This rule identifies known Domain IOCs (present in watchlist) in the DNS query logs. An alert will be created if DNSQuery matches with any value present in the Watchlist. To tune the query to your environment configure it using the 'DNS_Solution_Domain_IOCs' Watchlist.\n\nThe rule utilize [ASIM](https://aka.ms/AboutASIM) normalization, and is applied to any source which supports the ASIM DNS schema"
+  "This rule identifies known Domain IOCs in the [DNS query](https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/dn800669(v=ws.11)#using-etw-consumers) logs. It evaluates DNS logs against 100+ known IOC's from the available Watchlist which helps to detect DNS traffic from any known compromised domains.\n\nIt utilizes [ASIM](https://aka.ms/AboutASIM) normalization and is applied to any source which supports ASIM DNS schema"
```

Comment:
1. The link does not seem relevant.

1. Whenever using a Microsoft Learn URL, remove the language part (/en-us in this case)

1. This actually does not depict DNS traffic from compromised domains. It detects an attempt to resolve the name a known compromised domain, which may indicate an attempt to communicate with this domain, but not over DNS.

Reply:
thanks Ofer. Used this URL https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-dns#query
Rephrased description.